pyproject.toml

```diff
@@ -89,6 +89,7 @@ test = "pytest"
 testpaths = ["tests"]
 python_files = ["*test_*.py"]
 addopts = "--strict-markers"
+filterwarnings = "error"
```

Comment:
will this make pytest fail at any len(warnings) > 0?

Reply:
Yep!